bigchaindb/common/transaction.py

```diff
@@ -410,7 +410,7 @@ class Transaction(object):
     TRANSFER = 'TRANSFER'
     GENESIS = 'GENESIS'
     ALLOWED_OPERATIONS = (CREATE, TRANSFER, GENESIS)
-    VERSION = bigchaindb.version.__version__
+    VERSION = bigchaindb.version.__short_version__[:-4]  # 0.9, 0.10 etc
```

Comment:
`[:4]` may be potentially brittle if we ever have alpha or beta (e.g. 0.9a1) releases in the future. Not sure if there's a better solution though (maybe only use the `<major>.<minor>` value?).